app/actor/PrivateRoomActor.scala

```diff
@@ -0,0 +1,14 @@
+package actor
+
+import akka.actor._
+
+object PrivateRoomActor {
+  def props: Props = Props[PrivateRoomActor]
+}
+
+class PrivateRoomActor extends WebSocketActor {
```

Comment:
This class is not needed change `PublicRoomActor` to more general `ChatRoomActor`